### PR TITLE
feat(input): add multi values input to replace multiline input

### PR DESCRIPTION
**Pull request by GermainBergeron**
### Proposed Changes

Add new multi-values input to replace non-connected one

### Potential Breaking Changes

None

### Acceptance Criteria

-   [x] The proposed changes are covered by unit tests
-   [x] The potential breaking changes are clearly identified
-   [x] [README.md](https://github.com/coveo/react-vapor/blob/master/README.md) is adjusted to reflect the proposed changes (if relevant)

### CODE REVIEW

File: packages/react-vapor/src/components/multiValueInputs/MultiValuesInput.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+                            const value = e.currentTarget.value;
+                            if (value !== '' && cData.isLast) {
+                                parentProps.addNewBox();
+                            }
+                        }}
+                        onBlur={(value: string) => {
+                            if (value === '' && !cData.isLast) {
+                                parentProps.removeBox(cData.id);
+                            }
+                        }}
```
Comment (rosalie-liu) [inline packages/react-vapor/src/components/multiValueInputs/MultiValuesInput.tsx]: Do we need `validate` and `validateOnChange`, also `labelTitle` and `labelProps` for this `InputConnected`?

File: packages/react-vapor/src/components/multiValueInputs/MultiValuesInput.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            </div>
+        ),
+    })
+)(MultilineBox);
+
+export interface MultiValuesInputProps {
+    id: string;
+    data: string[];
+}
+
```
Comment (rosalie-liu) [inline packages/react-vapor/src/components/multiValueInputs/MultiValuesInput.tsx]: I'm not sure if it's out of scope. `RemoveButton` appears if the input is not the last, but do we need an `AddButton` for the last one? 
image

Comment (GermainBergeron) [reply packages/react-vapor/src/components/multiValueInputs/MultiValuesInput.tsx]: The input is added as soon as the value is not empty so it's ok to not have the add button

File: packages/react-vapor/src/components/multiValueInputs/MultiValuesInput.tsx (hunk 1 above)
Comment (GermainBergeron) [reply packages/react-vapor/src/components/multiValueInputs/MultiValuesInput.tsx]: I will add an `inputProps` to add everything that is missing

**Comment by dmgauthier** [comment]
I think we should we remove the part that use the multiline box directly in the input examples, since your new component seem to remove the need of it. :clap: :clap: :+1:

**Comment by GermainBergeron** [comment]
> I think we should we remove the part that use the multiline box directly in the input examples, since your new component seems to remove the need for it. 👏 👏 👍

It's a bit different though, my component only allows one input per "line" but the box exemple has 3 inputs per "line"

**Pull request merged**